WordPress/Classes/ViewRelated/Media/MediaProgressCoordinatorNoticeViewModel.swift

```diff
@@ -5,6 +5,8 @@ enum MediaNoticeUserInfoKey {
 }
 
 struct MediaProgressCoordinatorNoticeViewModel {
+    static let ErrorNoticeTag: Notice.Tag = "MediaProgressCoordinatorNoticeViewModel.tag"
```

Comment:
Can we change this to class+reason to be consistent with the way we defined other tags like:
"WPError.Networking"
or 
ReachabilityUtils.NoConnection"
?

Reply:
done!